src/app/routes/loan/routes/identity/routes/identity-otp/components/otp-editable-phone/otp-editable-phone.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { RouteApiService } from 'src/app/routes/loan/shared/store/api/route-api.service';
+import { RouteUiService } from '../../shared/store/ui/route-ui.service';
+
+@Component({
+  selector: 'app-otp-editable-phone',
+  templateUrl: './otp-editable-phone.component.html',
+  styleUrls: ['./otp-editable-phone.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OtpEditablePhoneComponent {
+  constructor(
+    protected loanApiService: RouteApiService,
+    protected routeUiService: RouteUiService,
+  ) {}
+
+  editPhoneNumber() {
+    this.routeUiService.editPhoneNumber();
+  }
```

Comment:
Just an alternative implementation but if the component isn't doing anything other than passing through methods you can consider making the service in the constructor public and going direct to that from the template.